packages/app-runtime/src/modules/AppRuntimeModule.ts

```diff
@@ -12,14 +12,9 @@ export interface AppRuntimeModuleConfiguration extends ModuleConfiguration {}
 export abstract class AppRuntimeModule<TConfig extends AppRuntimeModuleConfiguration = AppRuntimeModuleConfiguration> extends RuntimeModule<TConfig, AppRuntime> {
     private readonly nativeEventSubscriptionIds: number[] = [];
 
-    protected subscribeToNativeEvent<TEvent>(event: Event, handler: EventHandler<TEvent>): void {
+    protected subscribeToNativeEvent<TEvent>(event: SubscriptionTarget<TEvent>, handler: EventHandler<TEvent>): void {
         const subscriptionResult = this.runtime.nativeEnvironment.eventBus.subscribe(event, handler);
-        if (subscriptionResult.isError) {
-            this.logger.error(subscriptionResult.error);
-            throw subscriptionResult.error;
-        }
-
-        this.nativeEventSubscriptionIds.push(subscriptionResult.value);
+        this.nativeEventSubscriptionIds.push(subscriptionResult);
```

Comment:
What is the reason for these changes?

Reply:
And this not only affects logging and throwing errors, but also lets you push a result and not its value?

Reply:
I don't understand :D

Reply:
it's only simplified